src/development/sic-calls/history.md

```diff
@@ -16,6 +18,65 @@
+- [@jasoriatanishq](https://x.com/jasoriatanishq) (Nethermind): finished the Pectra rebase and will then work on integrating the tree conversion. He also plans to work on the implementation of the Binary Tree EIP.
+- [@kt2am1990](https://x.com/kt2am1990) (Besu): rebased Verkle on Pectra and tested the conversion on Hoodi. While doing so, he identified some bugs and performance issues that are being fixed. Thomas keeps working on stateless verification.
+- [@GabRocheleau](https://x.com/GabRocheleau) for @EFJavaScript: working on passing all the tree conversion test vectors, generalizing the conversion to any target tree, and doing other architectural adjustments.
+
+### 2. *FatNet* state-growth testnet
+
+[@CPerezz19](https://x.com/CPerezz19) explained the goal of *FatNet*: replicate Nethermind’s idea with *PerfNet* but focus on state size growth and its implications for EL clients. He shared a [collaborative document](https://hackmd.io/@CPerezz/ryATkZIelx) to identify valuable metrics to gather.
+
+[@_sophiagold_](https://x.com/_sophiagold_) suggested including snap-sync metrics, and [@CPerezz19](https://x.com/CPerezz19) mentioned it is useful but requires chatting with Pari to figure out how to do this correctly, since it might have some networking assumptions.
+
+[@gballet](https://x.com/gballet) mentioned that EL clients already have a cap on the number of connected nodes, so we might be able to simulate this realistically. Additionally, he mentioned that some transaction load (and proper access patterns) should be happening on the chain to test the healing phase correctly.
+
+### 3. EIP-7748 – special-account conversion rule
+
+A follow-up discussion clarified some confusion in the last SIC regarding custom rules in EIP-7748 regarding not converting some data.
+
+[@gballet](https://x.com/gballet) confirmed that the accounts previously described in EIP-7748 don’t exist on mainnet. The new proposed action targets accounts described in [EIP-7610](https://eips.ethereum.org/EIPS/eip-7610): accounts with zero nonce, empty code, non-zero balance, and non-empty storage. There are accounts satisfying this condition on mainnet, and it will be helpful if we can clean them up during the conversion. The intention is to keep the invariant that if an account has non-empty storage, the nonce is greater than zero.
+
+It was pointed out that this rule doesn’t conflict with EIP-7702, since any activated delegation implies an increase of the nonce, thus stopping the `nonce==0` condition of the rule.
+
+EIP-7748 was already updated with this new rule, so EL clients are expected to update their implementation. This also means that execution-spec-tests should cover this new rule.
+
+### 4. EIP-7702 – delegation removal semantics
+
+[@ignaciohagopian](https://x.com/ignaciohagopian) wanted to discuss more precisely a topic tangentially touched on in the previous SIC call: which is the proper action in the Overlay Tree when an EIP-7702 delegation is removed?
```

Comment:
just realized that "overlay" only refers to the conversion, when this rule works after the conversion, too. So we should say "post-transition tree" here imo.

Reply:
or maybe just "converted tree"?

Reply:
Agree. Will change to "new tree".